Makefile.work

```diff
@@ -175,7 +175,6 @@ DOCKER_BUILD = docker build --no-cache \
 
 SONIC_BUILD_INSTRUCTION :=  make \
                            -f slave.mk \
-                           BLDENV=$(BLDENV) \
```

Comment:
>BLDENV=$(BLDENV) \ [](start = 27, length = 18)

I see it is used in these places. Do you want to remove them?

```
Makefile:12:    EXTRA_DOCKER_TARGETS=$(notdir $@) BLDENV=stretch make -f Makefile.work stretch
Makefile:14:    BLDENV=buster make -f Makefile.work $@
Makefile:34:    BLDENV=stretch make -f Makefile.work $@
Makefile:36:    BLDENV=buster make -f Makefile.work $@
```

Reply:
those are not inside the docker, cannot be removed.